cmake/incubator-test.cpp

```diff
@@ -1,11 +1,9 @@
 #define CAF_TEST_NO_MAIN
 
-#include "caf/fwd.hpp"
-#include "caf/test/dsl.hpp"
-#include "caf/test/unit_test_impl.hpp"
-#include "caf/type_id.hpp"
+#include "incubator-test.hpp"
 
 int main(int argc, char** argv) {
-  caf::init_global_meta_objects<>();
+  caf::core::init_global_meta_objects();
```

Comment:
For future reference: It's actually better to call this *after* the `init_global_meta_objects` below, because initializing the largest block first makes sure you allocate memory only once.